src/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -40,9 +40,6 @@
     <Compile Include="$(CommonPath)\System\Collections\Generic\LargeArrayBuilder.cs">
       <Link>Common\System\Collections\Generic\LargeArrayBuilder.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\System\Diagnostics\CodeAnalysis\ExcludeFromCodeCoverageAttribute.cs" Condition="'$(TargetGroup)'=='net463' or '$(TargetGroup)' == 'netcore50' or '$(TargetGroup)' == 'netcore50aot'">
```

Comment:
The condition on this seems to be a legacy of different changes keeping the restrictions of the previous, unnecessarily, and there's no such restriction elsewhere AFAICT, but if there's anywhere I've got something wrong, it's here.